proxy/mempool/mempool_schedule.py

```diff
@@ -408,6 +408,11 @@ def add_tx(self, tx: MPTxRequest) -> MPTxSendResult:
         sender_pool = self._get_or_create_sender_pool(tx.sender_address)
         LOG.debug(f'Got sender pool {tx.chain_id, tx.sender_address} with {sender_pool.len_tx_nonce_queue} txs')
 
+        if self.tx_cnt >= self._capacity:
+            top_tx = sender_pool.top_tx
```

Comment:
`top_tx` is the tx with the `lowest nonce` in the `sender_pool`.
`state_tx_cnt` is the `transaction counter` in the Neon on-chain state.

Examples of valid `top_tx`:
- state_tx_cnt = 5, sender_pool.queue_nonces = [10, 11, 12] -> top_tx.nonce = 10
- state_tx_cnt = 5, sender_pool.queue_nonces = [5, 6, 7] -> top_tx.nonce = 5

At the same time, `MPSenderTxPool` has property `pending_nonce`
Examples of valid `pending_nonce`:
- state_tx_cnt = 5, sender_pool.queue_nonces = [10, 11, 12] -> sender_pool.pending_nonce = 5
- state_tx_cnt = 5, sender_pool.queue_nonces = [5, 6, 7] -> sender_pool.pending_nonce = 7

So, `pending_nonce` is the last nonce in the queue without gaps with the `transaction counter` in the Neon state.

You should use `pending_nonce` in this place instead of `top_tx`

Reply:
🤦‍♂️ my mistake. Ok, will fix. Thanks